README.md

```diff
@@ -1 +1,33 @@
-# contrib
\ No newline at end of file
+# Repository for OpenVINO's extra modules
+
+This repository is intended for the development of so-called "extra" modules, contributed functionality. New modules quite often do not have stable API, and they are not well-tested. Thus, they shouldn't be released as a part of official OpenVINO distribution, since the library maintains backward compatibility, and tries to provide decent performance and stability.
+
+So, all the new modules should be developed separately, and published in the `openvino_contrib` repository at first. Later, when the module matures and gains popularity, it is moved to the central OpenVINO repository, and the development team provides production-quality support for this module.
+
+## How to build OpenVINO with extra modules
+You can build OpenVINO, so it will include the modules from this repository. Contrib modules are under constant development and it is recommended to use them alongside the master branch or latest releases of OpenVINO.
+
+Here is the CMake command for you:
+
+```sh
+$ cd <openvino_build_directory>
+$ cmake -DIE_EXTRA_MODULES=<openvino_contrib>/modules <openvino_source_directory>
+$ cmake --build . -j8
+```
+
+As the result, OpenVINO will be built in the <openvino_build_directory> with all modules from `openvino_contrib` repository. If you don't want all of the modules, use CMake's `BUILD_<module_name>` boolean options. Like in this example:
```

Comment:
What if "If you don't want all of the modules" -> "To disable specific modules"?

Reply:
Agree, fixed